### PR TITLE
Snapping scale limit doc

### PR DESCRIPTION
**Pull request by obrix**
Small documentation for the scale range limit feature for snapping.

* Add a paragraph to describe the feature and how it works
* Update the advanced mode snapping settings screenshot

### CODE REVIEW

File: docs/user_manual/working_with_vector/editing_geometry_attributes.rst [hunk 1]
```diff
@@ -165,6 +165,43 @@ proximity.
...
+  is. This is the default mode.
+* :guilabel:`Global`: Snapping is limited and only enabled when the current
+  scale of the map is between a global minimum and a global maximum value.
+* :guilabel:`PerLayer`: Snapping is limited and only enabled when the current
+  scale of the map is between a minimum and a maximum value each one of these
+  defined for each layer.
+
+
+When selecting :guilabel:`Global` mode two widgets become available
+to configure the minimum and maximum scales between which snapping is enabledS.
```
Comment (DelazJ) [inline docs/user_manual/working_with_vector/editing_geometry_attributes.rst:197]: Could we move this to the global section above? To have all details of the option in one place.

File: docs/user_manual/working_with_vector/editing_geometry_attributes.rst [hunk 2]
```diff
@@ -165,6 +165,43 @@ proximity.
...
+
+When selecting :guilabel:`PerLayer` mode two columns become available to
+configure the minimum and maximum scales for each layer.
```
Comment (DelazJ) [inline docs/user_manual/working_with_vector/editing_geometry_attributes.rst:200]: Same as above, for "per layer"

File: docs/user_manual/working_with_vector/editing_geometry_attributes.rst [hunk 3]
```diff
@@ -165,6 +165,43 @@ proximity.
...
+
+Please note that the minimum and maximum scales follow the qgis convention :
+minimum scale is the most "zoom out" scale while maximum scale is the most "zoomed in".
```
Comment (DelazJ) [inline docs/user_manual/working_with_vector/editing_geometry_attributes.rst:203]:
```suggestion
minimum scale is the most "zoomed out" scale while maximum scale is the most "zoomed in".
```
?

File: docs/user_manual/working_with_vector/editing_geometry_attributes.rst [hunk 4]
```diff
@@ -165,6 +165,43 @@ proximity.
...
+  defined for each layer.
+
+
+When selecting :guilabel:`Global` mode two widgets become available
+to configure the minimum and maximum scales between which snapping is enabledS.
+
+When selecting :guilabel:`PerLayer` mode two columns become available to
+configure the minimum and maximum scales for each layer.
+
+Please note that the minimum and maximum scales follow the qgis convention :
```
Comment (DelazJ) [inline docs/user_manual/working_with_vector/editing_geometry_attributes.rst:202]:
```suggestion
Please note that the minimum and maximum scales follow QGIS convention:
```

File: docs/user_manual/working_with_vector/editing_geometry_attributes.rst [hunk 5]
```diff
@@ -165,6 +165,43 @@ proximity.
...
+
+To limit snapping to a scale range you have three modes available :
+
+* :guilabel:`Disabled`: Snapping is enabled whatever the current map scale
+  is. This is the default mode.
+* :guilabel:`Global`: Snapping is limited and only enabled when the current
+  scale of the map is between a global minimum and a global maximum value.
+* :guilabel:`PerLayer`: Snapping is limited and only enabled when the current
+  scale of the map is between a minimum and a maximum value each one of these
+  defined for each layer.
```
Comment (DelazJ) [inline docs/user_manual/working_with_vector/editing_geometry_attributes.rst:193]: I understand what you mean but wonder if we can find a better formulation. Sorry. No inspiration, yet ;(

File: docs/user_manual/working_with_vector/editing_geometry_attributes.rst (hunk 1 above)
Comment (DelazJ) [inline docs/user_manual/working_with_vector/editing_geometry_attributes.rst:197]:
```suggestion
to configure the range of scales in which snapping is enabled.
```
?

**Review by DelazJ — COMMENTED** [review]
Thanks @obrix 
I suggest some restructuring that my text suggestions do not necessarily take into account.
Also note that the screenshot doesn't need a margin around; the CSS of our theme already takes care of this.

File: docs/user_manual/working_with_vector/editing_geometry_attributes.rst [hunk 6]
```diff
@@ -165,6 +165,43 @@ proximity.
...
+
+
+When selecting :guilabel:`Global` mode two widgets become available
+to configure the minimum and maximum scales between which snapping is enabledS.
+
+When selecting :guilabel:`PerLayer` mode two columns become available to
```
Comment (havatv) [inline docs/user_manual/working_with_vector/editing_geometry_attributes.rst:199]:
```suggestion
When selecting :guilabel:`Per layer` mode two columns become available to
```

File: docs/user_manual/working_with_vector/editing_geometry_attributes.rst [hunk 7]
```diff
@@ -165,6 +165,43 @@ proximity.
...
+
+To limit snapping to a scale range you have three modes available:
+
+* :guilabel:`Disabled`: Snapping is enabled whatever the current map scale
+  is. This is the default mode.
+* :guilabel:`Global`: Snapping is limited and only enabled when the current
+  scale of the map is between a global minimum and a global maximum value.
+* :guilabel:`Per layer`: Snapping is limited and only enabled when the current
+  scale of the map is between a minimum and a maximum value each one of these
+  defined for each layer.
```
Comment (havatv) [inline docs/user_manual/working_with_vector/editing_geometry_attributes.rst:193]:
```suggestion
* :guilabel:`Per layer`: Snapping is enabled on a per layer basis.
  The napping scale range is defined for each layer, as explained below.
```

**Review by havatv — APPROVED** [review]
Looks fine to me.

File: docs/user_manual/working_with_vector/editing_geometry_attributes.rst (hunk 1 above)
Comment (obrix) [reply docs/user_manual/working_with_vector/editing_geometry_attributes.rst]: Yes, done in last commit.

File: docs/user_manual/working_with_vector/editing_geometry_attributes.rst (hunk 2 above)
Comment (obrix) [reply docs/user_manual/working_with_vector/editing_geometry_attributes.rst]: Yes, done in last commit.

File: docs/user_manual/working_with_vector/editing_geometry_attributes.rst (hunk 7 above)
Comment (obrix) [reply docs/user_manual/working_with_vector/editing_geometry_attributes.rst]: I integrated your suggestion.

File: docs/user_manual/working_with_vector/editing_geometry_attributes.rst (hunk 1 above)
Comment (DelazJ) [reply docs/user_manual/working_with_vector/editing_geometry_attributes.rst]: Thanks

File: docs/user_manual/working_with_vector/editing_geometry_attributes.rst [hunk 8]
```diff
@@ -165,6 +165,41 @@ proximity.
...
+To limit snapping to a scale range you have three modes available:
+
+* :guilabel:`Disabled`: Snapping is enabled whatever the current map scale
+  is. This is the default mode.
+* :guilabel:`Global`: Snapping is limited and only enabled when the current
+  scale of the map is between a global minimum and a global maximum value.
+  When selecting this mode two widgets become available
+  to configure the range of scales in which snapping is enabled.
+* :guilabel:`Per layer`: Snapping is enabled on a per layer basis.
+  The snapping scale range is defined for each layer.
```
Comment (DelazJ) [inline docs/user_manual/working_with_vector/editing_geometry_attributes.rst:194]:
```suggestion
* :guilabel:`Per layer`: The snapping scale range is defined for each layer.
```
Snapping on a layer basis is already set when switching to advanced config, so available in the three modes.

Comment (obrix) [reply docs/user_manual/working_with_vector/editing_geometry_attributes.rst:194]: Done !

**Pull request merged**